cloudformation/database/serverless-database.yaml

```diff
@@ -0,0 +1,345 @@
+          - MinCapacity
+          - MaxCapacity
+      -
+        Label:
+          default: "Network Configuration"
+        Parameters:
+          - Vpc
+          - PrivateSubnetOne
+          - PrivateSubnetTwo
+    ParameterLabels:
+      Vpc:
+        default: "Which VPC should this be deployed to?"
+Parameters:
+  AdminPasswordSecret:
+    Description: Name or ARN of SecretManager secret for Admin Password.
+    Type: String
+    MinLength: 2
+    MaxLength: 1024
+  ClusterAdmin:
+    Description: Database Cluster Admin User Name.
+    Type: String
+    Default: clusteradmin
+    MinLength: 3
+    MaxLength: 64
+  Vpc:
+    Description: VPC ID to create the Database Cluster in (leave BLANK to create a new one).
+    Type: String
+    # Type: AWS::EC2::VPC::Id
+    Default: ""
+  PrivateSubnetOne:
+    Description: Subnet ID of the first subnet (leave BLANK to create).
+    Type: String
+    # Type: AWS::EC2::Subnet::Id
+    Default: ""
+  PrivateSubnetTwo:
+    Description: Subnet ID of the second subnet (leave BLANK to create). This subnet should be in different availability zone from PrivateSubnetOne.
+    Type: String
+    # Type: AWS::EC2::Subnet::Id
+    Default: ""
+  MinCapacity:
+    Description: Minimum Scaling Capacity for Database Cluster. Must be less than or equal to MaxCapacity.
+    Type: Number
+    Default: 1
+    AllowedValues:
+      - 1
+      - 2
+      - 4
+      - 8
+      - 16
+      - 32
+      - 64
+      - 128
+      - 256
+  MaxCapacity:
+    Description: Maximum Scaling Capacity for Database Cluster. Must be greater than or equal to MinCapacity
+    Type: Number
+    Default: 4
+    AllowedValues:
+      - 1
+      - 2
+      - 4
+      - 8
+      - 16
+      - 32
+      - 64
+      - 128
+      - 256
+Transform: AWS::Serverless-2016-10-31
+Conditions:
+  CreateVPC: !Equals [!Ref Vpc, '']
+  CreateSubnets: !Or [!Condition CreateVPC, !Equals [!Ref PrivateSubnetOne, ''], !Equals [!Ref PrivateSubnetTwo, '']]
+Rules:
+  EmptyVpcAssertions:
+    RuleCondition: !Equals
+      - !Ref Vpc
+      - ''
+    Assertions:
+      - Assert: !Equals
+          - !Ref PrivateSubnetOne
+          - ''
+        AssertDescription: PrivateSubnetOne must be empty if Vpc is not provided.
+      - Assert: !Equals
+          - !Ref PrivateSubnetTwo
+          - ''
+        AssertDescription: PrivateSubnetTwo must be empty if Vpc is not provided.
+Resources:
+  #
+  # Optional Networking
+  #
+  ServerlessDatabaseClusterVpc:
+    Type: 'AWS::EC2::VPC'
+    Condition: CreateVPC
+    Properties:
+      CidrBlock: 10.0.0.0/16
+      EnableDnsHostnames: true
+      EnableDnsSupport: true
+      InstanceTenancy: default
+      Tags:
+        - Key: Name
+          Value: ServerlessDatabaseClusterVPC
+        - Key: 'parallel-cluster:accounting'
+          Value: vpc
+        - Key: 'parallel-cluster:accounting:scheduler'
+          Value: slurm
+        - Key: 'parallel-cluster:accounting:version'
+          Value: '1.0'
```

Comment:
Here and below: I think we can avoid all tags, but Name.
I'm curious: why do you think they may be useful?

Reply:
I think I am trying to make sure the resources created by this stack are easily discernible in their relevant consoles, but maybe this ins't a good idea?